misk-core/src/main/kotlin/misk/sampling/RateLimiter.kt

```diff
@@ -107,6 +108,35 @@ class RateLimiter private constructor(
     }
   }
 
+  /**
+   * Returns the maximum number of permits that could have
+   * been acquired by a call to [tryAcquire], assuming the caller
+   * passed the same [timeout] and [unit].
+   *
+   * @return the permits remaining.
```

Comment:
nit: I think this `@return` line is redundant and implied by the function name and could be removed, thanks for the above description though!